openpype/pipeline/delivery.py

```diff
@@ -162,8 +164,41 @@ def deliver_single_file(
     if not os.path.exists(delivery_folder):
         os.makedirs(delivery_folder)
 
-    log.debug("Copying single: {} -> {}".format(src_path, delivery_path))
-    _copy_file(src_path, delivery_path)
+    if has_renumbered_frame:
+        src_dir = os.path.dirname(src_path)
+        src_filepaths = os.listdir(src_dir)
+        src_collections, remainders = clique.assemble(src_filepaths)
+
+        src_collection = src_collections[0]
```

Comment:
I know we are doing it like so elsewhere, but as I am looking at it, I think this is rather dangerous - we assume that the sequence we want to renumber is the first collection, but there is no guarantee.

In ideal world, there should be just one collection and no reminder - and in that case it should work as expected. But in case there are more collections... Like for example in case of in-place publishing where you can have whatever content there can be?

We should use the files from representation - or at least issue warning that there is more than one sequence there? But the use of files from the selected representation would be the safest way probably. What do you think @iLLiCiTiT @moonyuet

Reply:
i see your point and it is indeed way safer to the files from representation for the sequence part, especially when the user wants to enable to use the renumber option for the single frame.

Reply:
I wanted to add that might as well keep it consistent with line `190`, whatever is that we end up going with.